MultiServer.py

```diff
@@ -431,9 +431,12 @@ def _load(self, decoded_obj: dict, game_data_packages: typing.Dict[str, typing.A
                              for player, loc_data in decoded_obj["er_hint_data"].items()}
 
         # load start inventory:
-        for slot, item_codes in decoded_obj["precollected_items"].items():
-            self.start_inventory[slot] = [NetworkItem(item_code, -2, 0) for item_code in item_codes]
-
+        for slot, items in decoded_obj["precollected_items"].items():
+            # TODO: remove conditionals in version 0.4.5
```

Comment:
```suggestion
            # TODO: remove conditionals after version 0.4.5 and reject loading old multidata
```
An async started with 0.4.4 should be playable until 0.4.6 is out.